docs/wiki/framework/attach-framework.md

```diff
@@ -0,0 +1,95 @@
+---
+layout: wiki
+title: Attach Framework
+description: Explains how to add items to ACE's Attach system.
+group: framework
+order: 0
+parent: wiki
+mod: ace
+version:
+  major: 3
+  minor: 17
+  patch: 0
+---
+
+## 1. Config Values
+### 1.1 Make item attachable
+
+An item can be added to ACEs "Attach to Vehicle" and "Attach to Self" system by adding the following config value to the ``CfgWeapons`` or ``CfgMagazines`` configs:
```

Comment:
```suggestion
An item can be added to the ACE Attach framework by adding the ``ACE_attachable``` property to a class in ``CfgWeapons`` or ``CfgMagazines``. The value must be the classname of a valid class in ``CfgVehicles``:
```